identity/src/main/java/com/android/identity/PresentationHelper.java

```diff
@@ -161,6 +159,7 @@ public class PresentationHelper {
     private byte[] mSelectedNfcFile;
     private boolean mUseTransportSpecificSessionTermination;
     private boolean mSendSessionTerminationMessage;
+    private AtomicBoolean mWaitingForDeviceResponseSent = new AtomicBoolean(false);
```

Comment:
What's up with this field? Seems to be only ever written, and I cannot see what it's used for.

Reply:
Removed.